.grit/patterns/collection_to_bool.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Collection To Bool
+---
+
+Replace constant collection with boolean in boolean contexts.
+
+```grit
+engine marzano(0.1)
+language python
+
+or {
+    `if $cond:`,
+    r"elif (.*):"($cond),
```

Comment:
Avoid regex, especially in top level patterns. It should be fine to have a disregarded if that we look inside.

```suggestion
    `if $_:
         $_
     elif $cond:
        $_`
```

Reply:
I used regex here because won't the if pattern only match the first `elif` in a chain of `elif`s? I thought with the regex solution I could match all of them.

Reply:
Ah ok. This is a good case for using the AST then:

```suggestion
    elif_clause(condition=$cond)
```